src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_UsingStatement.cs

```diff
@@ -293,7 +293,14 @@ private BoundStatement RewriteUsingStatementTryFinally(SyntaxNode syntax, BoundB
             
             if ((!(methodOpt is null)) || Binder.TryGetSpecialTypeMember(_compilation, SpecialMember.System_IDisposable__Dispose, syntax, _diagnostics, out methodOpt))
             {
-                disposeCall = BoundCall.Synthesized(syntax, disposedExpression, methodOpt);
+                if (methodOpt.IsExtensionMethod)
```

Comment:
Consider using a `? :` operator here. If it extends to multiple lines, our formatting rule is each token on a new line e.g.,

```C#
var expr = cond
    ? true-consequence
    : false-consequence;